### PR TITLE
Fix Ansible documentation in part of example formatting. Part 1

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#examples-block

Per playbook best practices, each example should include a name: line:
```
EXAMPLES = r'''
- name: Ensure foo is installed
  modulename:
    name: foo
    state: present
'''
```
The name: line should be capitalized and not include a trailing dot.

**I'm gonna fix all modules in next PRs**

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

**Comment by Andersson007** [comment]
cc @acozine @samccann

File: plugins/modules/cloud/google/gcdns_record.py [hunk 1]
```diff
@@ -133,46 +133,46 @@
 '''
 
 EXAMPLES = '''
```
Comment (Akasurde) [inline plugins/modules/cloud/google/gcdns_record.py:135]: Shouldn't this module to move to `google.gcp` ?

Comment (felixfontein) [reply plugins/modules/cloud/google/gcdns_record.py:135]: @gundalow do you know?

File: plugins/modules/cloud/digital_ocean/digital_ocean.py [hunk 2]
```diff
@@ -115,17 +115,18 @@
...
     ssh_pub_key: 'ssh-rsa AAAA...'
     api_token: XXX
 
-# Create a new Droplet
-# Will return the droplet details including the droplet id (used for idempotence)
-
-- digital_ocean:
+- name: >
+    Create a new Droplet.
+    Will return the droplet details including the droplet id (used for idempotence)
```
Comment (felixfontein) [inline plugins/modules/cloud/digital_ocean/digital_ocean.py:128]: I would make this a comment. It does not belong in the task name IMO.
```suggestion
  # Will return the droplet details including the droplet id (used for idempotence)
```

File: plugins/modules/storage/zfs/zpool_facts.py [hunk 3]
```diff
@@ -43,13 +43,14 @@
...
+  zpool_facts: pool=rpool
 
-# Gather space usage about all imported ZFS pools
-- zpool_facts: properties='free,size'
+- name: Gather space usage about all imported ZFS pools
+  zpool_facts: properties='free,size'
 
-- debug: msg='ZFS pool {{ item.name }} has {{ item.free }} free space out of {{ item.size }}.'
+- name: Print gathered information
+  debug: msg='ZFS pool {{ item.name }} has {{ item.free }} free space out of {{ item.size }}.'
```
Comment (felixfontein) [inline plugins/modules/storage/zfs/zpool_facts.py:53]:
```suggestion
  debug:
    msg: 'ZFS pool {{ item.name }} has {{ item.free }} free space out of {{ item.size }}.'
```

File: plugins/modules/cloud/google/gcdns_record.py (hunk 1 above)
Comment (Andersson007) [reply plugins/modules/cloud/google/gcdns_record.py:135]: I don't know, let's fix the doc, then we decide:)

File: plugins/modules/cloud/digital_ocean/digital_ocean.py (hunk 2 above)
Comment (Andersson007) [reply plugins/modules/cloud/digital_ocean/digital_ocean.py]: good poing! thanks, done

File: plugins/modules/storage/zfs/zpool_facts.py (hunk 3 above)
Comment (Andersson007) [reply plugins/modules/storage/zfs/zpool_facts.py]: done, thanks!

File: plugins/modules/cloud/digital_ocean/digital_ocean.py (hunk 2 above)
Comment (felixfontein) [reply plugins/modules/cloud/digital_ocean/digital_ocean.py]: I meant making it a comment inside the task, not before. Comments outside are usually for multiple tasks.

File: plugins/modules/cloud/google/gcdns_record.py (hunk 1 above)
Comment (Andersson007) [reply plugins/modules/cloud/google/gcdns_record.py:135]: maybe they are made not by google

Comment (Andersson007) [reply plugins/modules/cloud/google/gcdns_record.py:135]: Copyright (C) 2015 CallFire Inc.

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing!

**Comment by Andersson007** [comment]
@felixfontein @Akasurde thanks for reviewing!

**Comment by Andersson007** [comment]
merged #332 into master

**Comment by felixfontein** [comment]
@Andersson007 thanks for fixing all these documentation issues!

**Pull request merged**